contracts/upgradeable_contracts/BasicRelativeDailyLimit.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./BasicLimits.sol";
+
+contract BasicRelativeDailyLimit is BasicLimits {
```

Comment:
Is it correct that here `BasicLimits` should be also be renamed to `AbsoluteDailyLimit`?

Reply:
Sorry, I forgot to push the changes with contract fixes that I made